template/razor_repl_expr.go

```diff
@@ -17,18 +17,25 @@ const (
 	literalTripleBackticks = "_=!TRIPLE_BT!=_"
```

Comment:
`@` and triple backticks can't go in the reservedKeywords slice below?

Reply:
I don't think so. Too risky to induce side effects IMO.